modules/goalign/cleanseqs/main.nf

```diff
@@ -0,0 +1,37 @@
+process GOALIGN_CLEANSEQS {
+    tag "goalign-cleanseqs:${aln}";
```

Comment:
I don't think this is necessary, since the name of the process is `GOALIGN_CLEANSEQS` so it would look like

`GOALIGN_CLEANSEQS (goalign-cleanseqs:aln1)`

```suggestion
    tag "${aln}";
```

Reply:
I removed the tag directive. As the tool takes as input a multi-sequence alignment (thus more than one sample), it does not take a meta parameter as input. Here, I followed what was done for the raxmlng  module. Which has similar input.